Makefile.core.mk

```diff
@@ -59,7 +59,7 @@ LD_EXTRAFLAGS += -X ${GO_MODULE}/pkg/istioversion.versionsFilename=${VERSIONS_YA
 IS_FIPS_COMPLIANT ?= false # set to true for FIPS compliance
 ifeq ($(IS_FIPS_COMPLIANT), true)
 	CGO_ENABLED = 1
-	LD_FLAGS = ${LD_EXTRAFLAGS} -s -w
+	LD_FLAGS = ${LD_EXTRAFLAGS} -tags strictfipsruntime -s -w
```

Comment:
I'm not sure but I think `strictfipsruntime` is specific to product builder so it should probably go to our fork

Reply:
okay, I will close this PR and create one in our fork.

Reply:
https://github.com/openshift-service-mesh/sail-operator/pull/255